pkg/command/utils/test.csv

```diff
@@ -0,0 +1,3 @@
+1, 2, 3
+4, 5, 6
+7, 8, 9
```

Comment:
suggest to move files for test purpose to ./test/asset folder (need to create one).